src/ImageProcessor.Web/Caching/IImageCache.cs

```diff
@@ -36,6 +36,11 @@ public interface IImageCache
         int MaxDays { get; set; }
 
         /// <summary>
+        /// Gets or sets the maximum number of days to cache the image in the browser.
+        /// </summary>
+        int BrowserMaxDays { get; set; }
```

Comment:
I doubt anyone is daft enough to build a custom cache that doesn't inherit from `ImageCacheBase` so I think we can go ahead and make the interface change. I'll make sure to note upon release and ensure all dependencies match.